docs/authentication/configuration/session-options.mdx

```diff
@@ -21,52 +21,45 @@ Fortunately, with Clerk you have to ability to fully control the lifetime of you
+3. Toggle the **Inactivity timeout** switch to enable it.
+4. Set your desired duration.
 
 ### Maximum lifetime
 
-Denotes the duration after which a session will expire and the user will have to sign in again, regardless of their activity on your site. By default, this setting is enabled with a default value of 7 days for all newly created instances. To find this setting and change the value, in the Clerk Dashboard, go to **[Sessions](https://dashboard.clerk.com/last-active?path=sessions)**.
+Denotes the duration after which a session will expire and the user will have to sign in again, regardless of their activity on your site.
+
+By default, this setting is enabled with a default value of 7 days for all newly created instances. To find this setting and change the value:
 
-<Images
-width={3024}
-height={1652}
-src="/docs/images/authentication/session-options/maximum-lifetime.jpg"
-alt="The 'Sessions' settings in the Clerk Dashboard with a red outline around the 'Maximum lifetime' section. This option is toggled on, revealing an input for customizing duration of a session lifetime."
-/>
+1. Navigate to the the Clerk Dashboard.
+2. Select your application, then select **[Sessions](https://dashboard.clerk.com/last-active?path=sessions)** from the navigation sidebar.
+3. Toggle the **Maximum lifetime** switch to enable it.
+4. Set your desired duration.
 
 ## Multi-session applications
 
-Multi-session applications allow users to be signed into more than one account at a time from the same browser. Users may switch the active account by opening the [`<UserButton />`](/docs/components/user/user-button) and selecting the desired account.
+One of the most powerful features that Clerk provides out of the box is multi-session applications.
 
-To enable this feature in your application, in the Clerk Dashboard, go to **[Sessions](https://dashboard.clerk.com/last-active?path=sessions)**. Turn on the **Mutli-session handling** toggle.
+A multi-session application is an application that allows multiple accounts to be signed in from the same browser at the same time. The user can switch from one account to another seamlessly. Each account is independent from the rest and has access to different resources.
 
-<Images
-width={3024}
-height={1652}
-src="/docs/images/authentication/session-options/multisession-handling.jpg"
-alt="The 'Multi-session handling' section under the 'Sessions' settings in the Clerk Dashboard. A red arrow is pointing to the 'Multi-session handling' toggle, which is toggled on, and there is also a red arrow pointing to the 'Apply Changes' button."
-/>
+To enable multi-session in your application, you need to configure it in the Clerk Dashboard.
 
-Learn more about working with multi-session applications with Clerk components and custom flows in our [detailed guide](/docs/custom-flows/multi-session-applications).
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=sessions)
```

Comment:
Same suggestions for consolidating these steps as above:
https://github.com/clerk/clerk-docs/pull/788/files#r1518068995